home.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
The first page in a website should be index.html even though you might call it your "home" page. You should rename this file to index.html.